docs/reference/data-streams/lifecycle/tutorial-manage-existing-data-stream.asciidoc

```diff
@@ -0,0 +1,136 @@
+[role="xpack"]
+[[tutorial-manage-existing-data-stream]]
+=== Tutorial: Update existing data stream
+
+preview::[]
+
+To update the lifecycle of an existing data stream you do the following actions:
+
+. <<set-lifecycle>>
+. <<delete-lifecycle>>
+
+[discrete]
+[[set-lifecycle]]
+==== Set a data stream's lifecycle
+
+To add or to change the retention period of your data stream you can use the <<data-streams-put-lifecycle, PUT lifecycle API>>.
+
+* You can set infinite retention period, meaning that your data should never be deleted. For example:
++
+[source,console]
+----
+PUT _data_stream/my-data-stream/_lifecycle
+{ } <1>
+----
+// TEST[setup:my_data_stream]
+<1> An empty payload means that your data stream is still managed but the data will never be deleted. Managing a time
+series data stream such as logs or metrics enables {es} to better store your data even if you do not use a retention period.
+
+* Or you can set the retention period of your choice. For example:
++
+[source,console]
+----
+PUT _data_stream/my-data-stream/_lifecycle
+{
+  "data_retention": "30d" <1>
+}
+----
+// TEST[continued]
+<1> The retention period of this data stream is set to 30 days. This means that {es} is allowed to delete data that is
+older than 30 days at its own discretion.
+
+The changes in the lifecycle are applied on all backing indices of the data stream. You can see the effect of the change
+via the <<data-streams-explain-lifecycle, explain API>>:
+
+[source,console]
+--------------------------------------------------
+GET .ds-my-data-stream-*/_lifecycle/explain
```

Comment:
This can use the data stream name itself, which would be less confusing
```suggestion
GET /my-data-stream/_lifecycle/explain
```

Reply:
I like the idea but currently this is not supported. I will create an enhancement issue for this.